src/openzaak/utils/schema.py

```diff
@@ -31,6 +32,30 @@ class use_ref:
 
 
 class AutoSchema(_AutoSchema):
+    def should_include(self):
+        if hasattr(self.view.serializer_class, "inclusion_serializers"):
```

Comment:
probably will need a little bit more love for dynamic serializers _if_ it becomes relevant

Reply:
How could we define/determine the possible inclusions for dynamic serializers? Could we attempt to derive them by following the serializer relations (like I did before?)